manager/orchestrator/replicated/slot.go

```diff
@@ -27,7 +27,21 @@ func (is slotsByRunningState) Less(i, j int) bool {
 		}
 	}
 
-	return iRunning && !jRunning
+	if iRunning && !jRunning {
+		return true
+	}
+
+	// Use Slot number as a tie-breaker. This will make the order
+	// of task termination deterministic when scaling down.
+	iSlot := is[i]
+	jSlot := is[j]
```

Comment:
We use `is[i]` and `is[j]` above, so it seems strange to assign shorthands to them here. For consistency, we should either define these at the beginning of the function, or use `is[i]` and `is[j]` directly. My preference would be the latter, since `is[i]` and `is[j]` are short and assigning them to a variable does not make the code clearer.